### PR TITLE
Optimized gym badges for retina displays

### PR DESCRIPTION
**Pull request by MikeDuister**
## Description
Optimized gym badges for retina screen & created a badge for uncontested gyms

## Motivation and Context
* Gym badges were not made for retina
* Uncontested gym had old icon

## How Has This Been Tested?
* With Chrome on Windows 10

## Screenshots (if appropriate):
screenshot_110

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by hemsleyk** [comment]
+1

**Comment by joveice** [comment]
+1

**Comment by AHAAAAAAA** [comment]
Good idea! The uncontested one is a bit too flashy for my liking and confusing, perhaps we should keep it grey and inconspicuous following the design of the original?

**Comment by joveice** [comment]
@AHAAAAAAA I think colors are best. but if people want that sure!

**Comment by MikeDuister** [comment]
@AHAAAAAAA Made it that way so that it would stand out as being available. Will have a new one ready tomorrow.

**Comment by joveice** [comment]
@AHAAAAAAA Oh, nvm, yea gray on tha would be cool. Don't mind my confusion :P

**Comment by parkershepherd** [comment]
+1 for gray/white uncontested. Icons look great though! Now just for the pokestops...

**Comment by jnovack** [comment]
Personally I like the PokeRev icons.  I don't need to see the logos, I need to see the colors.  A single icon that is different colors is more quickly recognizable than an logo in a color.  The problem is that the logo is a hi-res logo that looks awful at that size.

File: static/map.js [hunk 1]
```diff
@@ -574,13 +574,18 @@ function setupPokemonMarker(item, skipNotification, isBounceDisabled) {
 }
 
 function setupGymMarker(item) {
+  var gymIcon = {
+	url: "static/forts/" + gym_types[item.team_id] + ".png",
+	scaledSize: new google.maps.Size(40, 40)
```
Comment (andrewda) [inline static/map.js]: Each indentation level should be 2 spaces. Please change lines 578 and 579 to comply :)

**Comment by andrewda** [comment]
@jnovack I'm torn between the two. These icons look fantastic, but I agree that they are a bit difficult to see at a glance. The PokeRev icons are more basic, but get the point across better, and are more easily distinguished. If there were high res PokeRev icons (or something similar), I would be in favor of those, but at the moment it's a real 50/50.

**Comment by jnovack** [comment]
Will an SVG do?

https://dribbble.com/shots/2831980-Pok-mon-GO-Team-Logos-Vector-Download

**Comment by parkershepherd** [comment]
I think the high contrast of the badges in the PR work a lot better against the busy background of a map, especially for some of the colorful/dark themes. I bet cutting out badge shapes from  the main wallpaper in @jnovack would look pretty cool too though.

Could do something similar to the pokemon icons where multiple themes are allowed, but that smells like another PR for later

**Pull request closed without merging**